src/harmony/Modal/Modal.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { CSSProperties, ComponentProps, HTMLAttributes, PropsWithChildren, useEffect, useMemo } from 'react';
+import { IconXOutline } from '@taskany/icons';
+import cn from 'classnames';
+
+import { useKeyboard, KeyCode } from '../../hooks/useKeyboard';
+import { Portal } from '../../components';
+import { nullable } from '../../utils';
+import { Button } from '../Button/Button';
+
+import s from './Modal.module.css';
+
+interface ModalProps extends React.HTMLAttributes<HTMLDivElement> {
+    visible?: boolean;
+    width?: string | number;
+
+    onClose?: () => void;
+    onShow?: () => void;
+}
+
+interface ModalCrossProps extends Omit<ComponentProps<typeof Button>, 'view'> {}
+
+export const ModalCross = ({ className, size = 's', ...props }: ModalCrossProps) => (
```

Comment:
add component usage to story

Reply:
added to story & recordings to description